apps/api/middleware/jwt_auth.py

```
@@ -47,18 +51,61 @@ class AuthClaims(TypedDict, total=False):
 
 AUTH_SECRET_ENV_VAR = "AUTH_SECRET"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "AUTH_SECRET_ENV_VAR"

(S105)